Datahub.Core/Services/IPowerBiDataService.cs

```diff
@@ -20,7 +20,7 @@ public interface IPowerBiDataService
         public Task<bool> DeleteDataset(Guid id);
         public Task<IList<PowerBi_Report>> GetAllReports();
         public Task<bool> AddOrUpdateCataloguedReports(IEnumerable<PowerBi_ReportDefinition> reportDefinitions);
-        public Task<PowerBi_Report> GetReportById(Guid id);
+        public Task<PowerBi_Report> GetReportById(Guid id, bool includeWorkspace = false);
```

Comment:
@gcalexk I try not to use default parameters on interfaces, this is a personal opinion but default parameters can lead to undesired effects.